pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java

```diff
@@ -200,6 +200,34 @@ public ConfigSuccessResponse updateSchema(
     return new ConfigSuccessResponse(successResponse.getStatus(), schemaAndUnrecognizedProps.getRight());
   }
 
+  @PUT
+  @Produces(MediaType.APPLICATION_JSON)
+  @Consumes(MediaType.APPLICATION_JSON)
+  @Path("/schemas/fixDateTimeFormat/{schemaName}")
+  @Authenticate(AccessType.UPDATE)
+  @ApiOperation(value = "Update date time format in schema", notes = "Update date time format in schema")
+  @ApiResponses(value = {
+      @ApiResponse(code = 200, message = "Successfully updated schema"),
+      @ApiResponse(code = 404, message = "Schema not found"),
+      @ApiResponse(code = 400, message = "Missing or invalid request body"),
+      @ApiResponse(code = 500, message = "Internal error")
+  })
+  public ConfigSuccessResponse fixDateTimeFormat(
+      @ApiParam(value = "Name of the schema", required = true) @PathParam("schemaName") String schemaName,
```

Comment:
We want to take the table name instead of the schema name. We can put it under the `PinotSegmentRestletResource`, and read the table config and schema for the table from the cluster.